public/app/features/dashboard/panel_editor/VizTypePickerPlugin.tsx

```diff
@@ -135,6 +136,13 @@ interface PanelPluginBadgeProps {
 const PanelPluginBadge: React.FC<PanelPluginBadgeProps> = ({ plugin }) => {
   const display = getPanelStateBadgeDisplayModel(plugin);
 
+  if (
+    plugin.signature &&
+    plugin.signature !== PluginSignatureStatus.valid &&
+    plugin.signature !== PluginSignatureStatus.internal
+  ) {
+    return <PluginSignatureBadge status={plugin.signature} />;
+  }
   if (plugin.state !== PluginState.deprecated && plugin.state !== PluginState.alpha) {
     return null;
   }
```

Comment:
nit: what if we had this logic inside a container element that wraps PluginSignatureBadge?

Reply:
Yeah, this is worth considering but the changes we are introducing here are going to change soon, so I would skip this for now.